containers/docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Note that this bash script is called `docker.sh`. In the future you may have `rkt.sh` or `aci.sh`. It resides in a agnostic folder named `containers`. While I did not create it, you have the future option to create and reference `versions` here. So a future version of quantopian dockerfile might be `containers/versions/10.0.0-dev/Dockerfile` (or .../Container.aci). Would be a worthy todo/enhancement item. I've been using this format for many of my projects as well as other repos I contribute to. The turnkey aspect really helps the new users in getting them up and running quicker.